### ISSUE TITLE
Any way to hide titlebar text? 

### ISSUE THREAD

**Issue by Skrell** [MrOtherGuy/firefox-csshacks#87]
I'm using Waterfox Classic, and hate that the tabs title shows up in the windows titlebar, is there any CSS that can hide this text permanently?

**Comment by mimecry** [comment]
try this maybe: https://addons.mozilla.org/en-US/firefox/addon/custom-titlebar-text/

**Comment by Skrell** [comment]
Nope, it lets me customize the text, but not eliminate it unfortunately.

**Comment by mimecry** [comment]
if you just need to eliminate the tab title, you can change the titlebar text to a blank space, but that won't get rid of the browser name. either way, it's not something that CSS is capable of affecting

**Comment by MrOtherGuy** [comment]
Yeah, @mimecry is correct - one cannot use css to modify and/or hide the text in titlebar.

But if you are trying to remove it then why not remove the whole titlebar? Firefox has a toggle to hide it in Customization view.

**Comment by Skrell** [comment]
Unfortunately I like having the title bar in general as it makes it easier to click and drag, but I guess this post answered my question... I know you USED to be able to do this but I guess something changed in Firefox/Waterfox along the way to make it impossible to completely hide.   What about changing the opacity of the font to 0 or something?   Is that  a possibility?

**Comment by MrOtherGuy** [comment]
> What about changing the opacity of the font to 0 or something? Is that a possibility?

No, you just cannot modify the titlebar since it is OS level feature. Firefox just tells the OS to render some text there as a window title, and the OS does the rest.

But, there is another toggle "extra drag space" in customization view to add extra padding above of tabs. Maybe that works for you. Or you could enable menubar (while hiding titlebar) so that render above tabs and you *can* modify the menubar appearance - except if you are using mac.

**Comment by Skrell** [comment]
Unfortunately I don't think Waterfox Classic has that option... Is it possible to use CSS to add this extra space between the tabs on top and the top of the window?

**Comment by MrOtherGuy** [comment]
Weird, the extra drag space option has existed for - well forever from probably from Firefox 29 or some such. And menubar didn't work for you either?

I mean, sure, you could use something like `#TabsToolbar{ margin-top: 14px }` or something but then your window controls won't be at the top anymore though you could fix that with `.titlebar-buttonbox-container{ margin-top: -14px }` I think.
